### PR TITLE
Fix Styles on Sample Project

### PR DESCRIPTION
**Pull request by dimonovdd**
### This fix changes the display of Pickers and Text Editors.
**Before this fix, it looked like this:**

| Picker | Entry | Editor |
|:------:|:-----:|:-----:|
|entry| entry|editor|

### CODE REVIEW

File: Samples/Samples/App.xaml [hunk 1]
```diff
@@ -11,19 +11,29 @@
                 <Setter Property="Visual" Value="{x:Static local:App.PreferredVisual}" />
             </Style>
             <Style TargetType="Editor">
-                <Setter Property="TextColor" Value="{OnPlatform iOS='Black'}" />
+                <Setter Property="TextColor" Value="Black" />
+                <Setter Property="Visual" Value="Default" />
```
Comment (mattleibow) [inline Samples/Samples/App.xaml:15]: I see you are resetting the visual. Do the colors not fix it?

**Review by mattleibow — COMMENTED** [review]
Do you have screenshots of the after?

File: Samples/Samples/App.xaml (hunk 1 above)
Comment (dimonovdd) [reply Samples/Samples/App.xaml:15]: Visual Material is not working very well. I don't have time to play and configure it in a sample project. I just want to be able to test Essentials API

**Comment by dimonovdd** [comment]
**After:**

| Picker | Entry | Editor |
|:------:|:-----:|:-----:|
|telegram-cloud-photo-size-2-5229176396197245787-y|telegram-cloud-photo-size-2-5229176396197245786-y|telegram-cloud-photo-size-2-5229176396197245785-y|

File: Samples/Samples/App.xaml (hunk 1 above)
Comment (dimonovdd) [reply Samples/Samples/App.xaml:15]: I do not see a problem in the design of this application, the main thing is that it is comfortable for us

**Comment by dimonovdd** [comment]
@mattleibow I also noticed problems on iOS with Dark mode. I fixed it, but did it in Light mode style.

|            | **Before** |            |
|:------:|:-----:|:-----:|
|Home|Share|Browser|
|            | **After** |            |
|Home|Share|Browser|

**Comment by dimonovdd** [comment]
Hi, @mattleibow 
Is there any problem with this PR? If there are problems, I am ready to fix them.

**Comment by jfversluis** [comment]
A good style is definitely important and sorry for not giving this the attention it deserves. I don't think at this point We want to invest too much time in this. I do very much appreciate the time and effort that you put into this though! Thank you!

**Pull request closed without merging**